spec/src/consensus.rs

```diff
@@ -24,10 +24,10 @@ pub(crate) const ORPHAN_RATE_TARGET_RECIP: u64 = 20;
 const MAX_BLOCK_INTERVAL: u64 = 60 * 1000; // 60s
 const MIN_BLOCK_INTERVAL: u64 = 8 * 1000; // 8s
 const TWO_IN_TWO_OUT_CYCLES: Cycle = 2_580_000;
-pub(crate) const EPOCH_DURATION_TARGET: u64 = 8 * 60 * 60 * 1000; // 8 hours
-pub(crate) const MAX_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MIN_BLOCK_INTERVAL; // 3600
-pub(crate) const MIN_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MAX_BLOCK_INTERVAL; // 480
-pub(crate) const GENESIS_EPOCH_LENGTH: u64 = 2_000;
+pub(crate) const EPOCH_DURATION_TARGET: u64 = 4 * 60 * 60 * 1000; // 4 hours
+pub(crate) const MAX_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MIN_BLOCK_INTERVAL; // 1800
+pub(crate) const MIN_EPOCH_LENGTH: u64 = EPOCH_DURATION_TARGET / MAX_BLOCK_INTERVAL; // 240
+pub(crate) const GENESIS_EPOCH_LENGTH: u64 = MAX_EPOCH_LENGTH;
```

Comment:
@zhangsoledad @nirenzang not sure how to set the genesis epoch length. The old value 2000 is out of range, so I just set it to `MAX_EPOCH_LENGTH`.